### ISSUE TITLE
Change license?

### ISSUE THREAD

**Issue by JarmoKukkola** [scola/Qart#5]
Could you, please, change license to MIT or Apache 2.0 to allow commercial usage?

Thanks for excellent work.

**Comment by scola** [comment]
@JarmoKukkola No, GPLv3 license do not limit commercial usage, you can use the project for commercial usage

**Comment by vipzrx** [comment]
@scola gpl要求修改后的代码开源，他们要apache和mit是想用你的代码，商用，但是不愿意开源
